profile/README.md

```diff
@@ -0,0 +1,65 @@
+# Metal³
+
+The Metal³ project (pronounced: "Metal Kubed") provides components for bare
+metal host management with Kubernetes. You can enroll your bare metal machines,
+provision operating system images, and then, if you like, deploy Kubernetes
+clusters to them. From there, operating and upgrading your Kubernetes clusters
+can be handled by Metal³. Moreover, Metal³ is itself a Kubernetes application,
+so it runs on Kubernetes, and uses Kubernetes resources and APIs as its
+interface.
+
+Metal³ is one of the providers for the Kubernetes sub-project [Cluster
+API](https://github.com/kubernetes-sigs/cluster-api). Cluster API provides
+infrastructure agnostic Kubernetes lifecycle management, and Metal³ brings the
+bare metal implementation.
+
+This is paired with one of the components from the OpenStack ecosystem,
+[Ironic](https://ironicbaremetal.org/) for booting and installing machines.
+Metal³ handles the installation of Ironic as a standalone component (there's no
+need to bring along the rest of OpenStack). Ironic is supported by a mature
+community of hardware vendors and supports a wide range of bare metal
+management protocols which are continuously tested on a variety of hardware.
+Backed by Ironic, Metal³ can provision machines, no matter the brand of
+hardware.
+
+In summary, you can write Kubernetes manifests representing your hardware and
+your desired Kubernetes cluster layout. Then Metal³ can:
+
+* Discover your hardware inventory
+* Configure BIOS and RAID settings on your hosts
+* Optionally clean a host's disks as part of provisioning
+* Install and boot an operating system image of your choice
+* Deploy Kubernetes
+* Upgrade Kubernetes or the operating system in your clusters with a
+  non-disruptive rolling strategy
+* Automatically remediate failed nodes by rebooting them and removing them from
+  the cluster if necessary
+
+You can even deploy Metal³ to your clusters so that you can manage other
+clusters.
+
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+## Contacts
+* `#cluster-api-baremetal` on [Kubernetes Slack](https://slack.k8s.io/)
+* Metal³ [development [mailing list](https://groups.google.com/g/metal3-dev)
+* Zoom community meeting call on Wednesdays at 14:00 GMT
+
+## CI
+* Prow: https://prow.apps.test.metal3.io/
```

Comment:
No bare links. Either put them in `<>` if you want the link to be visible, if not then 

```suggestion
* Prow: <https://prow.apps.test.metal3.io/>
```
or
```suggestion
* [Prow](https://prow.apps.test.metal3.io/)
```